vcbuild.bat

```diff
@@ -165,6 +165,9 @@ if "%target%"=="Clean" echo deleting %~dp0deps\icu
 if "%target%"=="Clean" rmdir /S /Q %~dp0deps\icu
 :no-depsicu
 
+call tools\msvs\find_python.cmd
```

Comment:
Maybe redirect to NUL, to avoid `ERROR: The system was unable to find the specified registry key or value.`?